### PR TITLE
docs: added few steps to the getting started documentation for face landmark app #101

### PR DESCRIPTION
**Pull request by michellrodriguez08**
I added the first few steps for the face landmark app for iOS and Android. I reached a small block but as soon as I get that resolved I will add the rest of the steps, just wanted to practice the adding before it's 'perfect'. I also fixed the android configuration bug that Charles mention in the video with the tags and misspells.

### CODE REVIEW

File: docsite/docs/api_pages/face-landmark-detection.md [hunk 1]
```diff
@@ -9,4 +9,121 @@ Detects the main points of the face and its expressions in real time.
...
+
+2. **Navigate to Your Project Directory:** Navigate to your React Native project directory on your environment.
+
+3. **Install React Native MediaPipe:** Run the following command to install React Native MediaPipe and its dependencies:
+
+```
+yarn add react-native-mediapipe react-native-vision-camera react-native-worklets-core
+```
+
+    If correctly you should see a **react-native-mediapipe** dropdown inside **node_modules**
```
Comment (cdiddy77) [inline docsite/docs/api_pages/face-landmark-detection.md:34]: this is not necessary. doing `yarn add` is something that react native developers do every day.

File: docsite/docs/api_pages/face-landmark-detection.md [hunk 2]
```diff
@@ -9,4 +9,121 @@ Detects the main points of the face and its expressions in real time.
...
+
+**You must change the main Android manifest file** not the debugging one
+
+:::
+
+3. **Terminal Commands:** In your terminal run the following commands
+
+```
+yarn add @shopify/react-native-skia react-native-reanimated
+```
```
Comment (cdiddy77) [inline docsite/docs/api_pages/face-landmark-detection.md:95]: Move this up to where we do the other `yarn add` stuff

**Review by cdiddy77 — CHANGES_REQUESTED** [review]
Just a few changes. This will be great!

**Comment by michellrodriguez08** [comment]
I believe I addressed all the feedback received. I also added the rest of the steps to create the face landmark app. I removed the tabs but in my opinion I feel like it looks a bit crowded but let me know if it makes sense to you or if I should look for a different approach. Thank you!

**Review by cdiddy77 — APPROVED** [review]
ty!

**Pull request merged**